src/components/Answer.jsx

```diff
@@ -1,41 +1,90 @@
 import React, { useState } from 'react'
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
 import IconButton from '@material-ui/core/IconButton'
+import EditIcon from '@material-ui/icons/Edit'
 import store from '../state/store/configureStore'
+import { useTranslation } from 'react-i18next'
 
 const Answer = ({ text, type, placeholder, questionKey }) => {
-  const [email, setEmail] = useState('')
+  const [inputValue, setInputValue] = useState('')
+  const [filled, setFilled] = useState(false)
+  const { t } = useTranslation()
 
   const setAnswer = (event) => {
     event.preventDefault()
-    
-    store.dispatch({
-      type: 'SET_ANSWERS',
-      payload: { key: questionKey, answer: email },
-    })
+    if (filled) {
+      setFilled(false)
+    } else {
+      setFilled(true)
+      store.dispatch({
+        type: 'SET_ANSWERS',
+        payload: { key: questionKey, answer: inputValue },
+      })
+    }
   }
 
   return (
     <div className='type-field-container'>
       <h3>{text}</h3>
       <div>
         <form onSubmit={(event) => setAnswer(event)}>
-          <input
-            data-cy='input'
-            type={type}
-            value={email}
-            required
-            onChange={(event) => {
-              setEmail(event.target.value)
-            }}
-            placeholder={placeholder}
-          />
+          {type === 'toggle-btn' ? (
+            <div className='radio-group'>
+              <input
+                disabled={filled}
+                type='radio'
+                id='office'
+                data-cy='office-btn'
+                value='office'
+                name='selector'
+                required
+                onChange={(event) => {
+                  setInputValue(event.target.value)
+                }}
+              />
+              <label data-cy='office-lable' for='office'>
+                {t('answer.officeLable')}
+              </label>
+              <input
+                disabled={filled}
+                type='radio'
+                id='open-space'
+                data-cy='open-space-btn'
+                value='open-space'
+                name='selector'
+                required
+                onChange={(event) => {
+                  setInputValue(event.target.value)
+                }}
+              />
+              <label data-cy='open-space-lable' for='open-space'>
+                {t('answer.openSpaceLable')}
```

Comment:
If / when we get to have a third type of input, we should try to extract them. For now, I think it's ok!

Reply:
definitely